lte/gateway/c/oai/include/mme_events.h

```diff
@@ -26,7 +26,7 @@ extern "C" {
 /**
  * Helper function to initiate AsyncEventdClient in its own thread
  */
-int event_client_init(void);
+void event_client_init(void);
```

Comment:
any reason to change the return type to void?

Reply:
there is no return value
```
void event_client_init(void) {
  init_eventd_client(); <- this also returns void
}
```

Reply:
It may be better to update these other functions so that they actually return a success or failure code. Do we want the services get started in case of initialization failure for this client? I see that CHECK_INIT_RETURN pattern is not used for this, but probably we should. @ssanadhya what do you think?

Reply:
@ulaskozat  @themarwhal , Since Eventd is not a mandatory service for MME to run, the MME startup should not be blocked if it is unable to talk to Eventd. Let's not use CHECK_INIT_RETURN here as it will assert if the initialization failed. But, it will be good to log whether MME is able to communicate with Eventd or not. @ardzoht can comment if we already have some logs/alerts to track whether events are being received from MME or not.

Reply:
If that check is needed, we can do it in a separate Pull Request